config/product/istio.js

```diff
@@ -40,7 +40,7 @@ export function init(store) {
 
   basicType([
     'networking.istio.io.envoyfilter',
-    'networking.istio.io.serviceentrie',
+    'networking.istio.io.serviceentry',
```

Comment:
Google says the former isn't a thing

Reply:
This, `security.istio.io.authorizationpolicie` --> 'security.istio.io.authorizationpolicy' below and another two from similar changes all relate to kube resource types. I've looked at all four and I think they're all correct